jx_sqlite/snowflake.py

```diff
@@ -249,7 +251,7 @@ def remove(self, column_name, column):
             Log.error("Logic error")
 
         self.map[column_name]=[c for c in self.map[column_name] if c != column]
-                
+
```

Comment:
The convention is not trailing whitespace. But if your IDE does not remove them, it will be a pain to do int manually.  https://www.python.org/dev/peps/pep-0008/